File: TypeScript/1672_RichestCustomerWealth.ts
//@ts-ignore
function maximumWealth(accounts: number[][]): number {
    let maxWealth: number = 0;
    for(let i = 0; i < accounts.length; i++){
        let wealth: number = 0;
        for(let k = 0; k < accounts[i].length; k++){
            wealth += accounts[i][k];
        }
        if(wealth > maxWealth){
            maxWealth = wealth;
        }
    }
    return maxWealth;
};

File: Java/1672_RichestCustomerWealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int maxWealth = 0;
        for(int i = 0; i < accounts.length; i++){
            int wealth = 0;
            for(int k = 0; k < accounts[i].length; k++){
                wealth += accounts[i][k];
            }
            if(wealth > maxWealth){
                maxWealth = wealth;
            }
        }
        return maxWealth;
    }
}